yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -36,17 +33,13 @@
                     {{ workflow.version_modified }}
                 </div>
 
-                <div id="graph{{ forloop.counter }}" class = workflow-image></div>
-
                 <div class = "workflow-details" id="description">
                     <b>Description: </b>{{ workflow.description }}
                 </div>
 
-                <div style = "text-align:center">
-                    <a class = "more-details-button" href="{% url 'detailed_workflow' workflow.id workflow.version_id %}"> View Details</a>
-                </div>      
+
                 <!-- <button class="details-button" onClick = "description_view()">View Description</button> -->
-            </li>
+            </div class="card">
```

Comment:
you can omit `class=card` on the closing element

Reply:
Gocha